upi/aws/cloudformation/06_cluster_worker_node.yaml

```diff
@@ -1,26 +1,26 @@
 AWSTemplateFormatVersion: 2010-09-09
-Description: Template for Openshift Cluster UPI Node Launch (EC2 worker instance)
+Description: Template for OpenShift Cluster User-Provisioned Node Launch (EC2 worker instance)
 
 Parameters:
   InfrastructureName:
     AllowedPattern: ^([a-zA-Z][a-zA-Z0-9\-]{0,26})$
     MaxLength: 27
     MinLength: 1
-    ConstraintDescription: Infrastructure name must be alphanumeric, start with a letter and a maximum of 27 characters
+    ConstraintDescription: Infrastructure name must be alphanumeric, start with a letter, and have a maximum of 27 characters.
     Description: A short, unique cluster ID used to tag nodes for the kubelet cloud provider.
     Type: String
   RhcosAmi:
-    Description: Current RHEL CoreOS AMI to use for boostrap
+    Description: Current Red Hat Enterprise Linux CoreOS AMI to use for boostrap.
     Type: AWS::EC2::Image::Id
   WorkerSubnet:
-    Description: The subnets (recommend private) to launch the master nodes into
+    Description: The subnets, recommend private, to launch the master nodes into.
     Type: AWS::EC2::Subnet::Id
   WorkerSecurityGroupId:
     Description: The master security group ID to associate with master nodes.
     Type: AWS::EC2::SecurityGroup::Id
   IgnitionLocation:
     Default: https://api-int.$CLUSTER_NAME.$DOMAIN:22623/config/worker
-    Description: Location to fetch bootstrap ignition from. (Recommend to use the autocreated ignition config location.)
+    Description: Location to fetch bootstrap ignition from. You can use the generated Ignition config file location.
```

Comment:
What is the generated location?  Is this just saying "you probably want to use the default"?  Is that worth saying?  Maybe it should just be:

```
Description: Location of the Ignition config.
```